metastore.go

```diff
@@ -228,6 +228,16 @@ func (tx *metatx) users() (a []*User) {
 	return
 }
 
+// adminUserExists returns whether at least 1 admin-level user exists.
```

Comment:
the user data should be cached in the `Server` object. I think it's best to push this method to there. Users should be loaded into memory when the server boots up